toolchain/check/testdata/class/generic/subst_into_decl.carbon

```diff
@@ -0,0 +1,216 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/class/generic/subst_into_decl.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/class/generic/subst_into_decl.carbon
+
+class Class(T:! type) {
```

Comment:
Compared with fail_todo_use.carbon, what do you think of combining the tests into the same file? They seem similar, although the other is exercising a little bit that apparently isn't implemented yet.

Reply:
Refactored into a test for just member access and a test for just initialization. I also noticed I didn't have any coverage for member access naming a static member function, which seemingly doesn't work yet. Added test coverage for that as a TODO.